### ISSUE TITLE
Libero finetuned policy executes but fails to complete tasks successfully.

### ISSUE THREAD

**Issue by marianophielipp** [Physical-Intelligence/openpi#315]
I have been following the fine-tuning example for pi0 using the Libero dataset. The policy loads correctly, serves without errors, and executes actions in the environment, but it does not successfully complete any tasks (seems "confused" based on video inspection).

Despite verifying the training pipeline and checkpoint loading, the policy does not appear to be utilizing the learned behaviors effectively. Do you have any insights into this situation?

Steps to Reproduce:

I followed the fine-tuning workflow as per the documentation:
	1.	Compute Normalization Stats: uv run scripts/compute_norm_stats.py --config-name pi0_libero_low_mem_finetune
	2. 	Fine-Tune the Policy: XLA_PYTHON_CLIENT_MEM_FRACTION=0.9 uv run scripts/train.py pi0_libero_low_mem_finetune --exp-name=libero_ft_low_mem_3
	3.     Serve the finetuned Policy: uv run scripts/serve_policy.py policy:checkpoint --policy.config=pi0_libero --policy.dir=checkpoints/pi0_libero_low_mem_finetune/libero_ft_low_mem_3/29999 
        4.     Evaluate the policy:  examples/libero/main.py

**Comment by marianophielipp** [comment]
I attach the loss on the finetuning task:

![Image](https://github.com/user-attachments/assets/5b0f0537-96ac-4332-a4cf-4f88413d797d)

**Comment by haohuanw** [comment]
@marianophielipp when serving the policy, you will also want to use `pi0_libero_low_mem_finetune` instead of `pi0_libero` so that the lora trained weights are loaded and being used.

we will follow up to add warnings when there are weights loaded that weren't being used in model during policy serving.

**Comment by marianophielipp** [comment]
Awesome ! Thank you @haohuanw !
